machineconfiguration/v1/types_machineconfignode.go

```diff
@@ -158,6 +158,78 @@ type MachineConfigNodeStatus struct {
 	// +kubebuilder:validation:MaxItems=32
 	// +optional
 	IrreconcilableChanges []IrreconcilableChangeDiff `json:"irreconcilableChanges,omitempty"`
+	// internalReleaseImage describes the status of the release payloads stored in the node.
+	// +optional
```

Comment:
I think this field also needs the `+openshift:enable:FeatureGate=NoRegistryClusterOperations`?

Also may be good to add a few more lines of description, something like:
```
When specified, an internalReleaseImage custom resource exists on the cluster, and the specified images will be made available on the control plane nodes. This field will reflect the actual on-disk state of those release images.
```